Replicator/IncomingRev+Blobs.cc

```diff
@@ -28,14 +28,36 @@ namespace litecore { namespace repl {
     static std::atomic_int sMaxOpenWriters {0};
 #endif
 
+//#define FETCH_DANGLING_BLOBS
```

Comment:
I don't feel like we need this to rollback to the previous logic.

Reply:
okay, I will deal with in the final version.